app/utils/boot/tests/test_boot_import.py

```diff
@@ -43,7 +43,8 @@ class TestParseBoot(unittest.TestCase):
 
     def setUp(self):
         logging.disable(logging.CRITICAL)
-        self.db = mongomock.Database(mongomock.Connection(), "kernel-ci")
+        self.db = mongomock.Database(mongomock.MongoClient(),
+                                     "kernel-ci", None)
```

Comment:
Ha, it's a shame there's no `_store=None` default value, but that's how the API is:
https://github.com/mongomock/mongomock/blob/develop/mongomock/database.py#L23

Reply:
Absolutely, that was actually my first thought